maui-toolkit-toc.html

```diff
@@ -16,6 +16,25 @@
         </ul>
     </li>
 	<li><a href="/maui-toolkit/Migration">Migrate from Syncfusion .NET MAUI</a></li>
+    <li>
+        SfCalendar
+        <ul>
+            <li><a href="/maui-toolkit/Calendar/overview">Overview</a></li>
+            <li><a href="/maui-toolkit/Calendar/getting-started">Getting Started</a></li>
+            <li><a href="/maui-toolkit/Calendar/migration">Migrate from Xamarin.Forms</a></li>
```

Comment:
Remove this Migration file as not needed , remove file and link

Reply:
Removed the Respective Files and Link with Latest Commit